demo/admin/src/products/generated/ProductForm.tsx

```diff
@@ -223,11 +223,12 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
                                 />
                             )}
                         </Field>
+
                         <DateField
                             variant="horizontal"
                             fullWidth
-                            name="availableSince"
-                            label={<FormattedMessage id="product.availableSince" defaultMessage="Available Since" />}
```

Comment:
It's strange that the generator generates a grid column for availableSince, but no form field. Is this intentional?

Reply:
I wouldn't invest too much time into the old admin-generator